params/flare.go

```diff
@@ -0,0 +1,268 @@
+	case c.chainID.Cmp(SongbirdChainID) == 0:
+		return common.HexToAddress("0x3A1b3220527aBA427d1e13e4b4c48c31460B4d91")
+	case c.chainID.Cmp(FlareChainID) == 0:
+		return common.HexToAddress("0x1000000000000000000000000000000000000001")
+	default:
+		return common.HexToAddress("0x1000000000000000000000000000000000000001")
+	}
+}
+
+func (c *FlareConfig) DefaultAttestors() []common.Address {
+	switch {
+	case c.chainID.Cmp(CostonChainID) == 0:
+		return costonDefaultAttestors
+	case c.chainID.Cmp(SongbirdChainID) == 0:
+		return songbirdDefaultAttestors
+	case c.chainID.Cmp(FlareChainID) == 0:
+		return flareDefaultAttestors
+	}
+
+	var defaultAttestors []common.Address
+	defaultAttestorList := os.Getenv(defaultAttestorEnv)
+	if defaultAttestorList != "" {
+		defaultAttestorEntries := strings.Split(defaultAttestorList, ",")
+		for _, defaultAttestorEntry := range defaultAttestorEntries {
+			defaultAttestors = append(defaultAttestors, common.HexToAddress(defaultAttestorEntry))
+		}
+	}
+
+	return defaultAttestors
+}
+
+func (c *FlareConfig) DefaultValidators() (*ValidatorsDefault, error) {
+	var weight uint64
+	var nodeIDs []string
+	var steps []Step
+	switch {
+
+	case c.chainID.Cmp(TestingChainID) == 0:
+		nodeIDs = testingNodeIDs
+		weight = testingValidatorWeight
+
+	case c.chainID.Cmp(CostonChainID) == 0:
+		nodeIDs = costonNodeIDs
+		weight = costonValidatorWeight
+		steps = []Step{
+			{Epoch: 1604, Cutoff: 4}, // go down to 4 default validators one week after hard fork
+			{Epoch: 1772, Cutoff: 3}, // go down to 3 default validators two weeks after hard fork
+		}
+
+	case c.chainID.Cmp(SongbirdChainID) == 0:
+		nodeIDs = songbirdNodeIDs
+		weight = songbirdValidatorWeight
+		steps = []Step{
+			{Epoch: 42, Cutoff: 15}, // go down to 15 default validators two weeks after main net launch
+			{Epoch: 44, Cutoff: 10}, // go down to 10 default validators four weeks after main net launch
+			{Epoch: 46, Cutoff: 5},  // go down to 5 default validators six weeks after main net launch
+		}
+
+	case c.chainID.Cmp(FlareChainID) == 0:
+		nodeIDs = flareNodeIDs
+		weight = flareValidatorWeight
+
+	default:
+		customValidators := os.Getenv(customValidatorsEnv)
+		if customValidators == "" {
+			return nil, fmt.Errorf("custom validators not set for non-standard network (chain: %s)", c.chainID)
+		}
+		nodeIDs = strings.Split(customValidators, ",")
+		weight = customValidatorWeight
+	}
+
+	if len(nodeIDs) == 0 {
+		return nil, fmt.Errorf("no default validators set")
+	}
+
+	validators := make(map[ids.ShortID]uint64, len(nodeIDs))
+	for _, nodeID := range nodeIDs {
+		validator, err := ids.ShortFromPrefixedString(nodeID, constants.NodeIDPrefix)
+		if err != nil {
+			return nil, fmt.Errorf("could not parse validator (nodeid: %s): %w", nodeID, err)
+		}
+		validators[validator] = weight
+	}
+
+	v := ValidatorsDefault{
+		validators: validators,
+		steps:      steps,
+	}
+
+	return &v, nil
+}
+
+type ValidatorsDefault struct {
```

Comment:
I am not sure why the business logic related to `ValidatorsDefault` had to move to `params` folder. I would have expected mostly the hardcoded stuff only to move in?